docs/adrs/0001-CRA-and-general-architecture.md

```diff
@@ -0,0 +1,50 @@
+<!-- Short descriptive title -->
+# UI Refresh General Architecture 
+
+<!-- Date -->
+*2021-10-19*
+
+<!-- Summary -->
+The new UI refresh needed basic decisions around infrastructure. Typescript/React was the framework desired. State management was also discussed; Redux was chosen due to it's focus on app-level state, and it's popularity. This ADR covers the architectural decisions made to support those frameworks.
+
+## Authors <!-- GitHub Username(s) -->
+@dauglyon
+
+## Status <!-- Status of this ADR -->
+Accepted
+
+## Alternatives Considered <!-- Short list of considered alternatives, should include the chosen path -->
+- Create React App (CRA)
+- Manual React Setup
+- Other react bootstraping solutions
+
+## Decision Outcome <!-- Summary of the decision -->
+Create React App (CRA) was chosen to bootstrap the repo. The typescript template will be used, and redux added post-hoc.
+
+## Consequences <!-- Summary of the decision -->
+Using CRA allows fast initial setup, but adds some "magic" to the repo. We felt this was a good tradeoff for initial development. CRA also offers `eject`[^1] which allows for it to be removed as a dependancy (though doing so also removes many of it's benefits, such as not maintaining the webpack config). There is also a large dev community for CRA. 
+
+## Pros and Cons of the Alternatives <!-- List Pros/Cons of each considered alternative -->
+
+### Create React App (CRA)
+- `+` Fast and easy setup
+- `+` Large dev community, with existing tools and documentation
+- `-` Low flexibility (if we run into a unique react issue we need to solve, we may need to `eject`[^1])
+- `+` Low flexibility (allows us to focus more on development over bikeshedding)
```

Comment:
I think it's a good idea to keep the pros and cons separate. I think these lines make sense in either order, so my preference is for `+++-` instead of `++-+`.